tests/integration/models/application.spec.coffee

```diff
@@ -41,91 +41,134 @@ describe 'Application Model', ->
 
 		describe 'balena.models.application.create()', ->
 
-			afterEach ->
-				balena.pine.delete
-					resource: 'application'
-
-			it 'should be able to create an application w/o providing an application type', ->
-				balena.models.application.create
-					name: 'FooBar'
-					deviceType: 'raspberry-pi'
-				.then ->
-					promise = balena.models.application.getAll()
-					m.chai.expect(promise).to.eventually.have.length(1)
+			describe '[read operations]', ->
```

Comment:
Grouped the readonly/rejected operations to save some time from unnecessary app deletions & re-creations.